cmd/single_signon_test.go

```diff
@@ -1,11 +1,10 @@
 package cmd_test
 
 import (
+	"github.com/NearlyUnique/awsSts/cmd"
 	"net/http"
 	"net/http/httptest"
 	"testing"
-
-	"github.com/NearlyUnique/awsSts/cmd"
```

Comment:
I thought it was more common to use std pks at the top then others?

Reply:
It was my first try to use go ;-)